package.json

```diff
@@ -6,8 +6,10 @@
     "setup": "yarn install && yarn allow-scripts && yarn postinstall",
     "android": "yarn uninstall:android && react-native run-android",
     "android:apk": "yarn update-tokens && yarn gradle assembleRelease && yarn uninstall:android && yarn android:load-apk",
-    "android:bundle": "./scripts/check-env.sh && yarn update-tokens && yarn gradle bundleRelease && open android/app/build/outputs/bundle/release/",
```

Comment:
Why are you removing `yarn update-tokens` ? We need it.

Reply:
It was from when I was testing the bundle script before I had env vars. I reverted this.